site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -199,12 +200,12 @@ rules take precedence:
 
 When the `interestaction` attribute is missing it will default to an auto state.
 
-| Interestee Element | `action` hint    | Event Type       | Behaviour                            |
-| :----------------- | :--------------- | :------------------------------------------------------ |
-| `<* popover=*>`    | (auto)           | `'interest'`     | Call `.showPopover()` on the invokee |
-| `<* popover=*>`    | `togglePopover`  | `'interest'`     | Call `.showPopover()` on the invokee |
-| `<* popover=*>`    | (auto)           | `'loseinterest'` | Call `.hidePopover()` on the invokee |
-| `<* popover=*>`    | `togglePopover`  | `'loseinterest'` | Call `.hidePopover()` on the invokee |
+| Interestee Element | `action` hint    | Event Type       | Behaviour                              |
+| :----------------- | :--------------- | :--------------- | :--------------------------------------|
+| `<* popover=*>`    | (auto)           | `'interest'`     | Call `.togglePopover()` on the invokee |
```

Comment:
Are these changes right?

Reply:
As if right now yes. It's based on feedback from Mason during code review